timestamp column typo in code snippet example

cmackeen opened the pull request.

cmackeen committed:
timestamp column typo in code snippet example

ritesh99rakesh commented:
Hi @cmackeen it is on purpose. We use `timestamp` inside the code for calculation purpose. We removed `s` from `timestamps` to make the code little smaller.
I think that is not a big issue. If the justification satisfies you then I suggest you close the PR. Otherwise, I welcome your input. Thanks.

cmackeen commented:
> Hi @cmackeen it is on purpose. We use `timestamp` inside the code for calculation purpose. We removed `s` from `timestamps` to make the code little smaller.
> I think that is not a big issue. If the justification satisfies you then I suggest you close the PR. Otherwise, I welcome your input. Thanks.

Well I copied the code snippet (from the readme) after cloning the repo, and it throws an error because column "timestamps" is not found. So I deleted the "s" and it ran. The code snippet should work out-of-the-box

ritesh99rakesh commented:
Thank you @cmackeen. Sorry for the mistake.
